tests/payload/test_manifest.py

```diff
@@ -0,0 +1,306 @@
+
+    return manifest_obj
+
+
+@pytest.fixture
+def manifest_dict():
+    return {
+        "version": "0.1.0",
+        "created_at": "2022-12-01T00:00:00+00:00",
+        "expires_at": "2100-01-01T00:00:00+00:00",
+        "metadata": {
+            "name": "Service1",
+            "description": "Description of Service1",
+            "version": "0.1.1",
+            "authors": [
+                "mf <mf@golem.network>",
+                "ng <ng@golem.network>",
+            ],
+            "homepage": "https://github.com/golemfactory/s1",
+        },
+        "payload": [
+            {
+                "platform": {
+                    "arch": "x86_64",
+                    "os": "linux",
+                    "os_version": "6.1.7601",
+                },
+                "urls": [
+                    "http://girepo.dev.golem.network:8000/"
+                    "docker-gas_scanner_backend_image-latest-91c471517a.gvmi",
+                ],
+                "hash": "sha3:05270a8a938ff5f5e30b0e61bc983a8c3e286c5cd414a32e1a077657",
+            }
+        ],
+        "comp_manifest": {
+            "version": "0.1.0",
+            "script": {
+                "commands": [
+                    "run .*",
+                ],
+                "match": "regex",
+            },
+            "net": {
+                "inet": {
+                    "out": {
+                        "protocols": [
+                            "http",
+                        ],
+                        "urls": [
+                            "http://bor.golem.network",
+                        ],
+                    },
+                },
+            },
+        },
+    }
+
+
+@pytest.fixture
+def manifest_dict_with_alias(manifest_dict):
+    manifest_dict["createdAt"] = manifest_dict.pop("created_at")
+    manifest_dict["expiresAt"] = manifest_dict.pop("expires_at")
+    manifest_dict["compManifest"] = manifest_dict.pop("comp_manifest")
+    manifest_dict["payload"][0]["platform"]["osVersion"] = manifest_dict["payload"][0][
+        "platform"
+    ].pop("os_version")
+
+    return manifest_dict
+
+
+@pytest.fixture
+def manifest_dict_with_no_optional_values(manifest_dict):
+    manifest_dict.pop("metadata")
+    manifest_dict["payload"][0]["platform"].pop("os_version")
+
+    return manifest_dict
+
+
+def test_manifest_to_dict_no_alias(manifest_obj, manifest_dict):
+    assert manifest_obj.dict() == manifest_dict
+
+
+def test_manifest_to_dict_with_alias(manifest_obj, manifest_dict_with_alias):
+    assert manifest_obj.dict(by_alias=True) == manifest_dict_with_alias
+
+
+def test_manifest_to_dict_missing_values(
+    manifest_obj_with_no_optional_values, manifest_dict_with_no_optional_values
+):
+    assert manifest_obj_with_no_optional_values.dict() == manifest_dict_with_no_optional_values
+
+
+def test_manifest_to_json(manifest_obj, manifest_dict):
+    manifest_obj_json = json.dumps(manifest_obj.dict())
+    manifest_dict_json = json.dumps(manifest_obj.dict())
```

Comment:
uh, shouldn't it be?

```suggestion
    manifest_dict_json = json.dumps(manifest_dict)
```

and hasn't the IDE or some linting tool indicated the unused `manifest_dict` ?